<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>660</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stackedWidget{	
	border-image: url(:/images/zp.PNG);
}；

#stackedWidget *{
	border-image: url( );
}
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_3">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QToolButton" name="fan_on_btn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>520</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;华文行楷&quot;;
color: rgb(255, 170, 0);
</string>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QToolButton" name="led_off_btn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>620</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;华文行楷&quot;;
color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>关灯</string>
     </property>
    </widget>
    <widget class="QToolButton" name="led_on_btn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>590</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;华文行楷&quot;;
color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>开灯</string>
     </property>
    </widget>
    <widget class="QToolButton" name="fan_off_btn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>550</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;华文行楷&quot;;
color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QToolButton" name="send_btn">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>380</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;华文行楷&quot;;
color: rgb(0, 213, 213);</string>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QToolButton" name="pwm_on_btn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>450</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;华文行楷&quot;;
color: rgb(132, 132, 0);</string>
     </property>
     <property name="text">
      <string>开蜂鸣器</string>
     </property>
    </widget>
    <widget class="QToolButton" name="pwm_off_btn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>450</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;华文行楷&quot;;
color: rgb(132, 132, 0);</string>
     </property>
     <property name="text">
      <string>关蜂鸣器</string>
     </property>
    </widget>
    <widget class="QToolButton" name="shuma_on_btn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>400</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;华文行楷&quot;;
color: rgb(132, 132, 0);
</string>
     </property>
     <property name="text">
      <string>开数码管</string>
     </property>
    </widget>
    <widget class="QToolButton" name="shuma_off_btn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>400</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;华文行楷&quot;;
color: rgb(132, 132, 0);</string>
     </property>
     <property name="text">
      <string>关数码管</string>
     </property>
    </widget>
    <widget class="QLabel" name="central_lab">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>180</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;华文行楷&quot;;
color: rgb(0, 213, 213);</string>
     </property>
     <property name="text">
      <string>中心号码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="central_edt">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>190</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;
background-color: rgb(183, 231, 231);</string>
     </property>
     <property name="text">
      <string>13800280500 </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rcv_edt">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>230</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;
background-color: rgb(183, 231, 231);</string>
     </property>
     <property name="text">
      <string>15608179669 </string>
     </property>
    </widget>
    <widget class="QLabel" name="rcv_lab">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>220</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;华文行楷&quot;;
color: rgb(0, 213, 213);</string>
     </property>
     <property name="text">
      <string>接收号码</string>
     </property>
    </widget>
    <widget class="QLabel" name="flag_lab">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>260</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;华文行楷&quot;;
color: rgb(0, 213, 213);</string>
     </property>
     <property name="text">
      <string>flag：1.火灾 2.求助 3.吃饭</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="flag_edt">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>290</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;
background-color: rgb(183, 231, 231);</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="send_lab">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>320</y>
       <width>321</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;华文行楷&quot;;
color: rgb(0, 213, 213);</string>
     </property>
     <property name="text">
      <string>选择：2.火灾 3.小偷</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="send_edt">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>350</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;
background-color: rgb(183, 231, 231);</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="T_lab1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(132, 132, 0);
background-color: rgb(0, 255, 255);
font: 75 18pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="T_lab">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;华文行楷&quot;;
color: rgb(132, 132, 0);
</string>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="shidu_lab1_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;华文行楷&quot;;
color: rgb(132, 132, 0);

</string>
     </property>
     <property name="text">
      <string>湿度</string>
     </property>
    </widget>
    <widget class="QLabel" name="shidu_lab1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(132, 132, 0);
background-color: rgb(85, 255, 255);
font: 75 18pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="light_lab1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;华文行楷&quot;;
color: rgb(132, 132, 0);
</string>
     </property>
     <property name="text">
      <string>亮度</string>
     </property>
    </widget>
    <widget class="QLabel" name="light_lab">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>270</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(132, 132, 0);
background-color: rgb(170, 255, 255);
font: 75 18pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="led_lab">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>590</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>灯</string>
     </property>
    </widget>
    <widget class="QLabel" name="fan_lab">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>510</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>风扇</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 26pt &quot;华文行楷&quot;;
color: rgb(227, 113, 170);</string>
     </property>
     <property name="text">
      <string>欢迎进入自动报警系统</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>530</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;华文行楷&quot;;
color: rgb(255, 170, 0);
</string>
     </property>
     <property name="text">
      <string> 风扇</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>600</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;华文行楷&quot;;
color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>灯</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_lab">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>560</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(223, 223, 167);
font: 75 italic 27pt &quot;Arial&quot;;
</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="date_lab">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>620</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(223, 223, 167);
font: 75 italic 20pt &quot;Arial&quot;;
</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QToolButton" name="exit_btn">
     <property name="geometry">
      <rect>
       <x>1150</x>
       <y>0</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(176, 176, 176);
font: 75 14pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>154</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;黑体&quot;;
color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string>自动检测</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>350</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;黑体&quot;;
color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string>主动控制</string>
     </property>
    </widget>
    <widget class="QPushButton" name="useMapButton">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>0</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>使用指南</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
